docs/content/docs/pub-sub-implementations.md

```diff
@@ -162,21 +160,34 @@ When implemented, HTTP publisher can be used as webhooks sender.
 | GuaranteedOrder | yes |  |
 | Persistent | no| |
 
-#### Configuration
+#### Subscriber configuration
```

Comment:
ExactlyOnceDelivery should be true:)

Reply:
988ec6531abbd023c5fcba55b153e3c8ba953f13